clang/include/clang/Driver/Options.td

```diff
@@ -2668,6 +2668,16 @@ def fsanitize_skip_hot_cutoff_EQ
 
 } // end -f[no-]sanitize* flags
 
+def fallow_runtime_check_skip_hot_cutoff_EQ
+    : Joined<["-"], "fallow-runtime-check-skip-hot-cutoff=">,
+      Group<f_clang_Group>,
+      Visibility<[ClangOption, CC1Option]>,
+      HelpText<"Exclude __allow_runtime_check for the top hottest code "
```

Comment:
It should be __builtin_allow_runtime_check
it's identifier, someone can search docs by exact string

Reply:
Done